python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -899,3 +936,72 @@ def getReadNoise(self, exposureMetadata, taskMetadata, ampName):
         self.log.warning("Median readout noise from ISR metadata for amp %s "
                          "could not be calculated." % ampName)
         return np.nan
+
+    def computeGaussianHistogramParameters(self, im1Area, im2Area, imStatsCtrl, mu1, mu2):
+        """Compute KS test for a Gaussian model fit to a histogram of the
+        difference image.
+
+        Parameters
+        ----------
+        im1Area : `lsst.afw.image.MaskedImageF`
+            Masked image from exposure 1.
+        im2Area : `lsst.afw.image.MaskedImageF`
+            Masked image from exposure 2.
+        imStatsCtrl : `lsst.afw.math.StatisticsControl`
+            Statistics control object.
+        mu1 : `float`
+            Clipped mean of im1Area (ADU).
+        mu2 : `float`
+            Clipped mean of im2Area (ADU).
+
+        Returns
+        -------
+        varFit : `float`
+            Variance from the Gaussian fit.
+        chi2Dof : `float`
+            Chi-squared per degree of freedom of Gaussian fit.
+        kspValue : `float`
+            The KS test p-value for the Gaussian fit.
```

Comment:
Could you comment in Notes on what values are expected for Gaussian distribution? Or what are the range of values considered in this code?